src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,110 @@
+package task;
+
+public class TaskList {
+    private final Task[] taskList;
+    private int taskCount = 0;
+
+    public TaskList(int size) {
+        taskList = new Task[size];
+    }
+
+    public void printTasks() {
+        System.out.println("Take a look, all the tasks you have here, so many, yuuuuuuge\n");
+        for (int i = 0; i < taskCount; i++) {
+            int temp = i + 1;
+            System.out.println(temp + ". " + taskList[i].printTask());
+        }
+    }
+
+    public Task getTask(int id) {
+        return taskList[id];
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    public void deleteTask(int taskNumber) {
+        Task tempTask = taskList[taskNumber];
+        try {
+            taskCount--;
+            System.out.print("I've deleted this task:\n" + tempTask.getTaskType() + tempTask.getStatusIcon() +
+                    tempTask.getDescription() + "\nYou have " + taskCount + " task(s) now!\n");
+            for (int i = taskCount; i < taskList.length - 2; i++) {
+                taskList[i] = taskList[i + 1];
+            }
+        } catch (NullPointerException | ArrayIndexOutOfBoundsException e) {
+            System.out.println("Invalid task specified!");
+            taskCount++;
+        }
+    }
+
+    public void markTask(int taskNumber) {
+        Task temp = taskList[taskNumber];
+        if (temp == null) {
+            System.out.println("You're trying to mark a task that DOESN'T EXIST, like bad people on JAN 6. " +
+                    "Some of the kindest and most lovely souls I've met");
+        } else {
+            temp.markAsDone();
+            System.out.println(temp.getTaskType() + temp.getStatusIcon() + temp.getDescription());
+            System.out.println("Ya boi Donald took the LIBERTY to mark this done:\n");
+        }
+    }
+
+    public void unmarkTask(int taskNumber) {
+        Task temp = taskList[taskNumber];
+        if (temp == null) {
+            System.out.println("Stop trying to unmark tasks like ILLEGAL ALIENS after" +
+                    " I'm president: NOT HERE!");
+        } else {
+            temp.markAsUndone();
+            System.out.println("Here's the task promised but not completed, just like the DEMS\n");
+            System.out.println(temp.getStatusIcon() + temp.getDescription());
+        }
+    }
+
+    public void addTask(String input) {
```

Comment:
The variable input is well-named as it clearly indicates that it holds user input. It is appropriately concise and descriptive.